src/scss/_colors.scss

```diff
@@ -1,15 +1,17 @@
-@include oColorsSetUseCase('o-header-services-top', 'background', 'cold-3');
+@include oColorsSetUseCase('o-header-services-top', 'background', 'black');
 @include oColorsSetUseCase('o-header-services-top', 'text', 'white');
 @include oColorsSetUseCase('o-header-services-top-link', 'text', 'white');
-@include oColorsSetUseCase('o-header-services-top-link-hover', 'text', 'grey-tint2');
+@include oColorsSetUseCase('o-header-services-top-link-hover', 'text', 'black-20');
 
 // default theme
 @include oColorsSetUseCase('o-header-services-top-title', 'text', 'white');
-@include oColorsSetUseCase('o-header-services-primary-nav', 'background', 'pink-tint4');
+@include oColorsSetUseCase('o-header-services-primary-nav', 'background', 'white-60');
```

Comment:
In the [o-colors migration guide](https://github.com/Financial-Times/o-colors/blob/master/migration-guide.md) it states that `pink-tint4` should migrate to `black-30`, how come we chose `white-60` instead?